towhee/hub/builtin/operators/milvus_search.py

```diff
@@ -67,6 +72,20 @@ def __call__(self, query: list):
                 if schema.dtype in (101, 100):
                     self.kwargs['anns_field'] = schema.name
 
+        if 'limit' not in self.kwargs:
+            self.kwargs['limit'] = 10
+
+        index_type = self.collection.indexes[0].params['index_type']
+        if 'param' not in self.kwargs:
```

Comment:
It's better to use a dict for the default parameters

Reply:
good idea and updated it.